docs/components/Disclosure/Disclosure.stories.mdx

```diff
@@ -31,7 +31,15 @@ to change but is not required to.
 ## Content guidelines
 
 - `title` should be informative and label the type of content grouped in the
-  body of the Disclosure
+  body of the Disclosure.
+  - This can either be a string or a React component. If a React component is
+    used (containing multiple children), it should be wrapped in a container
+    element and not a `<Fragment>` to maintain correct UI styling.
+  - The elements passed as a `title` can be any heading content, plain text, or
```

Comment:
I'm wondering if this reads like a contradiction at all. we first say to basically use a `div`. then we say correct elements to use are phrasing content, which a `div` is not. after playing around with this a bit I think I understand it, but it's not super obvious.

what do you think?

similarly, I'd restructure this point to make it a bit clearer. right now it says

>The elements passed as a `title` can be any heading content, plain text, or HTML that can be used within a paragraph (AKA phrasing content) to ensure that the title is accessible and can be properly read by screen readers.

except that heading content will have the opposite effect

so instead, how about...

>The elements used within `title` can be plain text, or valid paragraph content HTML (AKA phrasing content) to ensure that the title is accessible and can be properly read by screen readers. A heading may also be used but will not be treated as a heading by assistive technologies.

or something along those lines.

Reply:
That sounds perfect! Thanks for the suggestion!